Paskab012/rails_budget_app
# Feature branch

#### Hello Reviewer ✋🏽 
## Description of changes made ♻️
- Rails project setup
- Configuration of linters errors
- Connection made with the Postgres database

#### Type of change

> **Please select the relevant options**


#### How Has This Been Tested?

> **Please describe the tests that you ran to verify your changes**


#### How should this be manually tested?

> **Provide the steps to manually test the functionalities implemented in this PR**

- After cloning the [repo](https://github.com/Paskab012/rails_budget_app.git)
- Check out this branch by `branch here`
- Run `yarn or npm install to install the project dependencies
- Run `rails s` to start the development server
- bundle install

#### Checklist


#### Screenshots

|        BEFORE                  |                     AFTER       |
| :----------------------------: | :-----------------------------: |
|                                | Note: You can even provide a    |
|                                | screenshot                      |
